### DISCUSSION TITLE
App added to Unraid CA

### DISCUSSION THREAD

**Discussion by ibracorp-sycotix** [Taxel/PlexTraktSync#622] [Show and tell]
Hi @Taxel 

Just letting you know I have created an app template for the Unraid CA store and am working on a IBRACORP video covering your app this weekend. 

If you have time, would you be able to look over the docs and let us know if we missed anything?
Can't seem to get an json created with the file mapping. 

It also gives you a chance to help us make the video showing the app correctly.
Further, you will be able to change your instructions for Unraid as the template will be there for everyone to use already. 

Let me know!

**Comment by glensc** [comment]
Seems incomplete. Nothing shown. i.e where's the outcome to be seen by people who read this post?
